### PR TITLE
Add openapi asyncio client

### PR DESCRIPTION
**Pull request by mrichard91**
Adds an asyncio client based on the OpenAPI generator output.

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```
Comment (MrPickles) [inline .github/workflows/python.yml]: 3.7 is already EOL. should we make the minimum version start with 3.8?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,199 @@
...
+Authentication schemes defined for the API:
+<a id="AuthToken"></a>
+### AuthToken
+
+- **Type**: API key
+- **API key parameter name**: Authorization
+- **Location**: HTTP header
+
+
+## Author
```
Comment (MrPickles) [inline README.md]: should we remove this? (if it'll keep getting autogenerated, then we can keep as-is)

**Comment by mrichard91** [comment]
Addressed versions and empty author

File: README.md [hunk 3]
```diff
@@ -1,15 +1,212 @@
...
+Python 3.7+
+
+## Installation & Usage
+### pip install
+
+If the python package is hosted on a repository, you can install directly using:
+
+```sh
+pip install git+https://github.com/stairwell-inc/stairwell-python.git
+<<<<<<< HEAD
```
Comment (MrPickles) [inline README.md]: please fix the merge conflict

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  - test
+
+.pytest:
+  stage: test
+  script:
+   - pip install -r requirements.txt
+   - pip install -r test-requirements.txt
+   - pytest --cov=stairwell_openapi_client
+
+pytest-3.7:
```
Comment (MrPickles) [inline .gitlab-ci.yml]: there are a bunch of python 3.7 artifacts littered here and there. can you remove all of them?

**Comment by mrichard91** [comment]
Removed all the dangling refs to 3.7

**Review by MrPickles — APPROVED** [review]

**Pull request merged**